ting_search.module

```diff
@@ -973,5 +973,15 @@ function ting_search_submit($form, &$form_state) {
  */
 function _ting_search_quote($string) {
   $cqlDoc = new TingSearchCqlDoctor($string);
-  return $cqlDoc->string_to_cql($string);
+  // Hold cql query for further changes.
+  $return = $cqlDoc->string_to_cql();
+
+  // Issue 1392 filter on agency if required.
+  $filter_me = variable_get('ting_search_filter_holdingsitem',0);
+  if ($filter_me === 1) {
+    $return .= ' and holdingsitem.agencyId=' . variable_get('ting_agency');
```

Comment:
The issue uses `"` around the agency id e.g. `holdingsitem.agencyid=”100200”` but this is left out here.

Is it in fact unnecessary?